src/encoding.rs

```diff
@@ -0,0 +1,6 @@
+#[derive(Debug)]
```

Comment:
We should derive `Eq` and `PartialEq` here as well. It doesn't hurt and because the type is exposed our users might need these.

Reply:
fixed